clock/scheduledJobs/queueStatsActivityLogs.js

```diff
@@ -0,0 +1,240 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * clock/scheduledJobs/queueStatsActivityLogs.js
+ *
+ * This file is for queue stats activity logs.
+ */
+
+const activityLogUtil = require('../../utils/activityLog');
+const queueTimeMillis95th = require('../../config').queueTimeMillis95th;
+const redis = require('redis');
+const rconf = require('../../config').redis;
+const bluebird = require('bluebird');
+const ZERO = 0;
+const ONE = 1;
+const TWO = 2;
+const FLOAT_TWO = 2.0;
+const PERCENTILE_95TH = 0.95;
+
+// bluebird promise for redis
+bluebird.promisifyAll(redis.RedisClient.prototype);
+bluebird.promisifyAll(redis.Multi.prototype);
+
+// Create Redis connection
+const client = redis.createClient(rconf.instanceUrl.realtimeLogging);
+
```

Comment:
instead of creating another client here, can you use the one for realTimeLogging defined in  * ./cache/redisCache.js ? The client from this file is already "promisifyed", so do not need to line 19, 27 and 28